README.md

```diff
@@ -124,17 +133,15 @@ case fetch_user(id) do
 end
 ```
 
-### Similar Libraries
-
-For reference.
-
-- [exceptional](https://github.com/expede/exceptional)
-- [elixir-monad](https://github.com/nickmeharry/elixir-monad)
-- [happy_with](https://github.com/vic/happy_with)
-- [monad](https://github.com/rmies/monad)
-- [ok_jose](https://github.com/vic/ok_jose)
-- [towel](https://github.com/knrz/towel)
-
-*Possible extensions to include implementing bind on structs so that errors can be better handled.
-Implement a catch functionality for functions that error.
-Implement existing monad library protocols so can extend similar DB functionality e.g. slick*
+## Additional External Links and Resources
+
+* Elixir Forum
+  * [OK v1 library](https://elixirforum.com/t/ok-elegant-error-handling-for-elixir-pipelines-version-1-0-released/1932/)
+* [Railway programming](http://www.zohaib.me/railway-programming-pattern-in-elixir/)
+* Similar Libraries
+  * [exceptional](https://github.com/expede/exceptional)
```

Comment:
Here is my comment that I forgot to post:
> Ah, that's a highly philosophical question :smile:, but the short-term answer is that I was already typing it when I noticed you had existing `-` bullets. I was lazy and figured it didn't matter at the time since it's markdown. I'll change it.

:laughing: 

No, no decent reason. I just personally use asterisks. Easier to type (although it does require a chord key gesture) and it looks more like a bullet. Probably goes back to the days of MS Word when they first started allowing lowercase `o` to be a bullet before markdown.